ale_linters/ruby/rubocop.vim

```diff
@@ -36,6 +37,10 @@ if !exists('g:ale_ruby_rubocop_options')
     let g:ale_ruby_rubocop_options = ''
 endif
 
+if !exists('g:ale_ruby_rubocop_executable')
+    let g:ale_ruby_rubocop_executable = 'rubocop'
+endif
+
 call ale#linter#Define('ruby', {
 \   'name': 'rubocop',
 \   'executable': 'rubocop',
```

Comment:
You should change this to `executable_callback` and add a function, so ALE which check if your override is executable.